### ISSUE TITLE
Users are prompted to recommit "Deleted by us" conflicts

### ISSUE THREAD

**Issue by spalger** [sqren/backport#219]
When backport tries to cherry-pick a PR, if the changes conflict and have a set of conflicts which are simply "deleted by us" backport will suggest that the used add them back to the repo. This is probably part of the reason the CODEOWNERS file has been added back to Kibana a handful of times and I think is the wrong behavior for this scenario. It would probably be most appropriate to just ask the user to resolve the issue:

![image](https://user-images.githubusercontent.com/1329312/88193368-70032580-cbf2-11ea-916b-49731068f9cc.png)

![image](https://user-images.githubusercontent.com/1329312/88193124-2581a900-cbf2-11ea-84d8-fe35c6033625.png)

**Comment by spalger** [comment]
Thinking about this more I'm guessing this is caused by the fact that backport tried to look for conflict markers in the files and doesn't spot any.

**Comment by spalger** [comment]
If I handle this issue by running

```sh
git rm .github/CODEOWNERS src/dev/code_coverage/ingest_coverage/team_assignment/ingestion_pipeline_painless.json
```

then come back to the backport script I have to click enter "to stage them", which feels weird too... I kinda wish that the UI would just start at the same place every time instead of possibly skipping the step which shows me the path to the repo and lets me hit enter to continue.

**Comment by sqren** [comment]
Released in [v5.5.3](https://www.npmjs.com/package/backport/v/5.5.3)
